src/MetadataTools.jl

```diff
@@ -91,11 +91,11 @@ function get_pkg(pkg_name::AbstractString;
     for dir in readdir(vers_path)
         ver_num = convert(VersionNumber, dir)
         ver_path = joinpath(vers_path, dir)
-        sha = strip(readstring(joinpath(ver_path,"sha1")))
+        sha = strip(read(joinpath(ver_path,"sha1"), String))
```

Comment:
need to raise min compat for this on 0.6

Reply:
Ah right, thanks